packages/popup/src/LionPopup.js

```diff
@@ -4,29 +4,44 @@ import { overlays, LocalOverlayController } from '@lion/overlays';
 export class LionPopup extends UpdatingElement {
   static get properties() {
     return {
-      position: {
-        type: String,
+      placementConfig: {
+        type: Object,
       },
     };
   }
 
+  get placementConfig() {
+    return this._placementConfig;
+  }
+
+  set placementConfig(config) {
+    this._placementConfig = {
+      ...this._placementConfig,
+      ...config,
+    };
+
+    if (this._controller && this._controller._popper) {
+      this._controller.updatePlacementConfig(this._placementConfig);
+    }
+  }
```

Comment:
Hmm actually that's kinda true and also there's nothing to rerender because there is no render() in this class. So perhaps we can and should remove it then :\? At the same time, for clarity's sake, it's nice that it conventionally shows the exposed property as a "user endpoint" like that.

Reply:
Leaving it in for now, as third party tools (like open-wc storybook helper), and maybe docs generation, and other stuff may take use of this.

Reply:
You can use getters and setters with or without having the property in `static get properties`. If you have it in `properties`, the element will trigger a render when the setter is called. Alternatively, you can call `this.requestUpdate()` in the setter, but in general, I think placing the property in `properties` is the preferred method if the element needs to re-render when the property changes.

For properties that do not trigger renders, getter and setter without having it in `properties` is fine.

Reply:
Our current DelegateMixin also creates getters and setters under the hood. From a technical perspective, there is no need to add them in the properties list. From a readability/'tooling alignment' perspective there is and that's why we want to readd them in properties in the next version of DelegateMixin